### DISCUSSION TITLE
Tasmota32-lvgl - black screen & time and date with 1Hz

### DISCUSSION THREAD

**Discussion by OperatorB** [arendst/Tasmota#18996] [Q&A] [answered]
Hey!

I am currently testing the new 13.0.0 release on my Sunon 3.5" resistive touch enabled device and I have a experience maybe someone can explain.

So, after flashing the released .bin file I applied my template and uploaded my .ini file (not including touch related config). After all I can see black screen with time and date displayed in the top left corner with tiny white fonts refreshing every 1 sec.
I tried to upload an autoexec.be file but no matter what is inside the black screen with the time and date is overlayed on top.

Did anyone had experience with similar situation?

**Comment by sfromis** [comment, marked as answer]
That time update sounds like you've configured a testing DisplayMode to provide that. Try `DisplayMode 0`

**Reply by OperatorB** [reply]
Yep, that was the issue. Thank you for the very quick answer!
